easybuild/easyconfigs/s/SQLite/SQLite-3.16.2-gimkl-2017a.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
+version = "3.16.2"
+
+homepage = 'http://www.sqlite.org/'
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'version': '2017a', 'name': 'gimkl'}
+
+source_urls = ['http://www.hwaci.com/sw/sqlite/%s' % year for year in ['2016', '2017']]
+version_str = ''.join(v.zfill(2) for v in version.split('.')).ljust(8,'0')[1:]
+sources = ['sqlite-autoconf-%s.tar.gz' % version_str]
+
+dependencies = [
+    ('libreadline', '6.3'),
```

Comment:
@valtandor Is there a reason why you're not including Tcl as a dependency, like it is in existing SQLite easyconfigs?